internal/consensus/common_test.go

```diff
@@ -1057,26 +1056,14 @@ func signDataIsEqual(v1 *types.Vote, v2 *cmtproto.Vote) bool {
 		bytes.Equal(v1.Extension, v2.Extension)
 }
 
-// TODO: get rid of duplication with internal/state/helpers_test.go.
-func makeState(nVals, height int, chainID string) (sm.State, map[string]types.PrivValidator) {
-	vals := make([]types.GenesisValidator, nVals)
-	privVals := make(map[string]types.PrivValidator, nVals)
-	for i := 0; i < nVals; i++ {
-		secret := []byte(fmt.Sprintf("test%d", i))
-		pk := ed25519.GenPrivKeyFromSecret(secret)
-		valAddr := pk.PubKey().Address()
-		vals[i] = types.GenesisValidator{
-			Address: valAddr,
-			PubKey:  pk.PubKey(),
-			Power:   1000,
-			Name:    fmt.Sprintf("test%d", i),
-		}
-		privVals[valAddr.String()] = types.NewMockPVWithParams(pk, false, false)
-	}
+func makeStateNilParams(nVals int, chainID string) (sm.State, map[string]types.PrivValidator) {
```

Comment:
To reduce the changes, call this one `makeState` and teh other one `makeStateWithCP`.

Reply:
Yes, it is possible. I didn't want to undo all renaming Sergio did, but your comment makes sense.